skimage/morphology/selem.py

```diff
@@ -29,18 +29,18 @@ def square(width, dtype=np.uint8):
     return np.ones((width, width), dtype=dtype)
 
 
-def rectangle(width, height, dtype=np.uint8):
+def rectangle(nrows, ncols, dtype=np.uint8):
```

Comment:
@FirefoxMetzger this also needs the new deprecated, keyword-only arguments mentioned in the earlier issues. The reason is that, even though the values work as positional arguments, users could have used:

```python
rectangle(width=5, height=8)
```

and that code would now break.

Reply:
> rectangle(width=5, height=8)

I actually didn't believe this was possible until I tried it in a shell xD In retrospect, it makes a lot of sense that something like this would be possible in python, but I never even conceptualized this and - accordingly - didn't quite understand why such a test would be necessary. 

I guess you always learn something new; thanks for that.